app/views/shared/menu/_signed_out.html.haml

```diff
@@ -1,5 +1,5 @@
 %li#login-link
   %a{"auth" => "login"}
-    %i.ofn-i_017-locked
+    %img{ src: "/assets/menu/icn-login.svg" }
```

Comment:
I dont know :-)
I'll leave this as is for now.

Reply:
fyi, these svgs are new icons from the new design.

Reply:
I think rebuilding an icon font file every time you want to change or add something can be a bit of a pain. Maybe it's more efficient in terms of serving the assets though?

Reply:
Ah, I didn't know they were new. That makes sense then.

Five years ago it was seen as more efficient to load all the little images in one font file. But it turned out to be a pain because nobody wanted to touch the font file which needs the special IcoMoon web app to be edited. I also think that there is a lot of value in lazy loading and that the new http2 protocol can serve multiple files really efficiently. The font file now adds unneeded complexity.